controllers/research/upcomingEventController.js

```diff
@@ -109,11 +109,27 @@ const deleteUpcomingEventByID = (req, res) => {
         .catch((err) => sendError(res, err));
 };
 
+// new function which will trigger the organized property
+
+const isEventOrganized = (req,res) => {
+    const id = req.query.id;
+
+    if (!validateID(id)) {
+        return sendError(res, `Invalid ID: ${id}`);
+    }
+     const current_date = new Date();
+     const Event_date = req.query.dateOfEvent;
+     if(current_date > Event_date){
+        req.query.organized = true;
```

Comment:
- This should be updated in the `res` object if you are calling only this function explicitly through an endpoint.
- Or, update the database directly if that is not the case